en-US/firefox/new/horizon.lang

```diff
@@ -123,9 +130,9 @@ Firefox for other platforms
 Download Firefox <span>for Windows</span>
 
 
-## TAG: firefox_new_other_platforms
-;Download Firefox <span>for OS X</span>
-Download Firefox <span>for OS X</span>
+## TAG: macos_branding
```

Comment:
No. The string `Download Firefox <span>for OS X</span>` should be behind the `firefox_new_other_platforms` tag, then we should have `Download Firefox <span>for OS X</span>` behind `macos_branding`.

Since we never exposed the first string, we can expose it directly behind the original tag `firefox_new_other_platforms`. 

Imagine a situation where locales translate the macOS strings, including this one, but not the others. We still couldn't show that string, could we?

TL;DR this should be
```
## TAG: firefox_new_other_platforms
;Download Firefox <span>for macOS</span>
Download Firefox <span>for macOS</span>
```

The rest of the file is OK.